docs/overview/elastic_executor.md

```diff
@@ -0,0 +1,37 @@
+---
+id: overview_elastic_executor
+title: "Elastic Executor"
+---
+
+In order to get the functional effect of executing a specified Elasticsearch request, you must call the `execute` method defined on it, which returns a `ZIO` that requires an `ElasticExecutor`, fails with a `Throwable` and returns the relevant value `A` for that request.
```

Comment:
`you must call the `execute` method defined on it`
This is no longer true right? We're no longer exposing an `execute` on the request type.
The options are either to use an instance of `Elasticsearch` or use the accessor method `Elasticsearch.execute`.